pylops_mpi/utils/deps.py

```diff
@@ -0,0 +1,41 @@
+__all__ = [
+    "nccl_enabled"
+]
+
+import os
+from importlib import import_module, util
+from typing import Optional
+
+
+# error message at import of available package
+def nccl_import(message: Optional[str] = None) -> str:
+    nccl_test = (
+        # TODO: setting of OS env should go to READme somewhere
```

Comment:
This is explained here in the documentation of PyLops: https://pylops.readthedocs.io/en/stable/gpu.html 

We could add a similar page to that of PyLops-MPI and in the README simply say we refer to the documentation for details on how to use NCCL.